lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -127,14 +127,18 @@ function dosomething_organ_donation_send_postal_code() {
  *
  */
 function dosomething_organ_donation_store_registration() {
+  // Grab this to get the sid
   $params = drupal_get_query_parameters();
 
+  // Grab this to get the uid
+  global $user;
+
   try {
-    if ($params['sid'] && $params['uid']) {
+    if ($params['sid'] && $user->uid) {
```

Comment:
It's added as a meta tag [https://github.com/DoSomething/phoenix/blob/e5be0dd592c52264f7465971c3c56591ed507aea/lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php#L21](here)